### ISSUE TITLE
[update] Nextcloud 26.0.0?

### ISSUE THREAD

**Issue by kn0wmad** [Start9Labs/nextcloud-wrapper#36]
## Upstream
- [ ] Update server to 26.0.0 (or latest 25.x.x if 26 is not determined to be prod-ready)
- [ ] Fix #38 - unblocking:
    - [ ] Face Recognition deps & config
- [ ] Investigate #33 
- [ ] Fix #39 
## Backups
- [ ] New backups procedures with rsync? (eOS v0.4.0 dependent)

**Comment by kn0wmad** [comment]
Closing in favor of GH Milestones, will checklist in PRs
